modules/gltf/gltf_document.cpp

```diff
@@ -2909,8 +2913,8 @@ Error GLTFDocument::_parse_meshes(Ref<GLTFState> state) {
 				mat3d->set_flag(BaseMaterial3D::FLAG_ALBEDO_FROM_VERTEX_COLOR, true);
 				mat = mat3d;
 			}
-
-			import_mesh->add_surface(primitive, array, morphs, Dictionary(), mat, mat.is_valid() ? mat->get_name() : String(), flags);
+			import_mesh->add_surface(primitive, array, morphs,
+					Dictionary(), mat, mat->get_name(), flags);
```

Comment:
Why was the is_valid check removed? The cases where mat is null still exist.

Reply:
Added check back.